### PR TITLE
Feat: Add date option while activating savings

### LINKED ISSUE #679: Activating Savings Account is not expecting date.
Through Android app, you can provide, Submit Date, Approve Date for Savings account. While Activating savings account , it won't expect any date and gets activated with just one click

### PR DESCRIPTION
**Pull request by tarun0**
Fix #679 

(Tested on one saving account. I'll do testing on other accounts after setting the backend on the localhost.)

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by therajanmaurya** [comment]
@tarun0 Your build is failing. Please the travis build log and check out this wiki and see the log again. you will get, how will you have to fix the PR.

**Comment by tarun0** [comment]
@therajanmaurya Yes, done.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingsaccountactivate/SavingsAccountActivatePresenter.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    @Override
+    public void attachView(SavingsAccountActivateMvpView mvpView) {
+        super.attachView(mvpView);
+    }
+
+    @Override
+    public void detachView() {
+        super.detachView();
+        mSubscriptions.unsubscribe();
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingsaccountactivate/SavingsAccountActivatePresenter.java]: @tarun0 use mSubscriptions.clear() instead

**Comment by tarun0** [comment]
@therajanmaurya Changed

**Pull request merged**